ethcore/light/src/cache.rs

```diff
@@ -186,7 +186,7 @@ mod tests {
 
 		{
 			let corpus_time = &mut cache.corpus.as_mut().unwrap().1;
-			*corpus_time = *corpus_time - Duration::from_secs(6 * 3600);
+			*corpus_time = *corpus_time - Duration::from_secs(5 * 3600);
```

Comment:
why is this change needed?

Reply:
It substracts overflow otherwhise (of course that is only happening on windows). That looks like a windows bug/inconsistency with duration.

Reply:
🤦‍♂️ please describe it in the comment